std/net/curl.d

```diff
@@ -186,7 +186,8 @@ version(unittest)
     import std.file : deleteme;
     import std.path : buildPath;
 
-    import std.socket : Address, INADDR_LOOPBACK, Socket, TcpSocket;
+    import core.sys.windows.winsock2 : INADDR_LOOPBACK;
+    import std.socket : Address, Socket, TcpSocket;
```

Comment:
`std.socket` privately imports `core.sys.windows.winsock2`, so `INADDR_LOOPBACK` cannt be selected.